lte/gateway/python/integ_tests/s1aptests/test_dedicated_bearer_activation_idle_mode.py

```diff
@@ -0,0 +1,161 @@
+"""
```

Comment:
can you trigger this with actual RAR to session manager? also can you add a verification that the flow rules are preserved after switching back to connected state?